ch_tools/chadmin/internal/system.py

```diff
@@ -7,9 +7,9 @@
 
 
 def validate_version(version: str) -> None:
-    pattern = r"^\d{2}\.\d{2}\.\d+\.\d+$"
```

Comment:
I see. The issue relates fort *.8.*.*
nit: we could just use `^\d{2}\.\d{1,2}\.\d+\.\d+$`